packages/client/src/ext/jwt-simple.ts

```diff
@@ -0,0 +1,213 @@
+/*
+ * jwt-simple
+ *
+ * JSON Web Token encode and decode module for node.js
+ *
+ * Copyright(c) 2011 Kazuhito Hokamura
```

Comment:
Can you replace these references with something like I did for the `asyncEventEmitter` code [here](https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/util/src/asyncEventEmitter.ts)

Reply:
done